introduction.md

```diff
@@ -1,3 +1,11 @@
 Hello, I'm Bart. I'm looking forward to working in the dojo.
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?